### PR TITLE
[20893] Prepare 3.0.x-devel to become master

### PR DESCRIPTION
**Pull request by EduPonz**
## Description

This PR prepares Shapes Demo Docs for Fast DDS 3.0.x-devel to become master

Related implementation PR:
* eProsima/ShapesDemo#127

## Contributor Checklist

- [x] Commit messages follow the project guidelines. 
- _N/A_: Documentation tests pass locally.
- _N/A_: Applicable backports have been included in the description.

## Reviewer Checklist

- [x] The PR has a milestone assigned.
- [x] The title and description correctly express the PR's purpose.
- [x] Check contributor checklist is correct.
- [x] CI passes without warnings or errors.

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/196/)

File: .github/workflows/mirror.yml [hunk 1]
```diff
@@ -3,16 +3,18 @@ on:
   push:
     branches:
       - 'master'
+      - '2.14.x'
 
 jobs:
-  mirror_job:
+  mirror_job_master:
```
Comment (elianalf) [inline .github/workflows/mirror.yml:9]:
```suggestion
  mirror_job_master:
  if: github.ref == 'refs/heads/master'
```

**Review by elianalf — COMMENTED** [review]
We should add in the README:

> [!WARNING]
> In preparation for v3.0.0, Fast DDS' master branch is undergoing major changes entailing **API breaks**.
> Until Fast DDS v3.0.0 is released, it is strongly advisable to use the latest stable branch, [2.14.x](https://github.com/eProsima/Shapes-Demo-Docs/tree/2.14.x).

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/199/)

**Review by elianalf — APPROVED** [review]

**Pull request merged**